models/users.js

```diff
@@ -0,0 +1,9 @@
+const users = (sequelize, DataTypes) =>
+  sequelize.define('users', {
+    displayName: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: DataTypes.INTEGER,
+    image: DataTypes.STRING,
+  });
+
+module.exports = users;
```

Comment:
```suggestion
const users = (sequelize, DataTypes) =>
  sequelize.define('users', {
    displayName: DataTypes.STRING,
    email: DataTypes.STRING,
    password: DataTypes.INTEGER,
    image: DataTypes.STRING,
  },
  {
    timestamps: false,
  });

module.exports = users;
```
Vi sua thread la no slack, tive o mesmo problema e o pessoal no plantão me ajudou, ai to deixando a sugestão aqui.
mais pelo que entendi por default ele pega os timestamps ai se vc não quiser vc tem que passar esse campo ai pra dizer pra ele que não quer os timestamps